agent_event_publisher_http/Cargo.toml

```diff
@@ -15,6 +15,7 @@ agent_verification = { path = "../agent_verification" }
 anyhow = "1.0"
 async-trait.workspace = true
 cqrs-es.workspace = true
+http-serde-ext = "1.0"
```

Comment:
This crate has only 19k downloads (all time). Have we considered using a more popular alternative (such as `http-serde`)? Or are there any extra features we require that I'm unaware of?

Reply:
Not sure anymore why I initially went with `http-serde-ext` because indeed `http-serde` also does the job :) 

I've replaced it now :+1: